public/kibi_timeline.js

```diff
@@ -235,18 +235,34 @@ define(function (require) {
                   const startValue = value;
                   const startRawValue = startRawFieldValue[i];
 
-                  const content =
+                  let content =
                       '<div title="index: ' + indexId +
                       ', startField: ' + params.startField +
-                      (params.endField ? ', endField: ' + params.endField : '') +
-                      '">' + labelValue +
+                      (params.endField ? ', endField: ' + params.endField : '') + '">' + labelValue +
                       (params.useHighlight ? '<p class="tiny-txt">' + timelineHelper.pluckHighlights(hit, highlightTags) +
                       '</p>' : '') + '</div>';
 
-                  let style = 'background-color: ' + groupColor + '; color: #fff;';
+                  let style = `background-color: ${groupColor}; color: #fff;`;
+                  if (!endFieldValue || startValue === endFieldValue[i]) {
+                    // here the end field value missing but expected
+                    // or start field value === end field value
+                    // force vis box look like vis point
+                    style = `border-style: none; background-color: #fff; color: ${groupColor}; border-color: ${groupColor}`;
+                    const divregex = /(<div.*>)(.*)(<\/div>)/g;
+                    const contentDivParts = divregex.exec(content);
```

Comment:
Instead of this, you can have an angular template where you would use `ng-class` to set the appropriate class. Then you would compile that template to get the final HTML. See tooltip of the tilemap vis in kibana for some example.